sb_vision/tokens.py

```diff
@@ -179,65 +37,68 @@ def __init__(self, id: int, certainty: float=0.0) -> None:
         """
         self.id = id
         self.certainty = certainty
+        self.pixel_coords = pixel_coords
+        self.pixel_centre = pixel_centre
 
     @classmethod
     def from_apriltag_detection(
-        cls,
-        apriltag_detection: 'ApriltagDetection',
-        image_size: Tuple[int, int],
-        distance_model: Optional[str],
+            cls,
+            apriltag_detection: 'ApriltagDetection',
+            camera_model: Optional[str],
     ) -> 'Token':
         """Construct a Token from an April Tag detection."""
+
+        pixel_coords = np.array([list(l) for l in apriltag_detection.p])
```

Comment:
Should this also be being converted to a `tuple`? (If so, can we make it a `typing.NamedTuple`?)